src/shared/components/species-name-parts-for-table/SpeciesScientificNameForTable.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Nothing special for the scientific name in the table.
+ * Just re-export it for convenience.
+ */
+import { ScientificName } from '../species-name-parts';
```

Comment:
Scientic_name is always italic. Should we add the class here?

Reply:
The imported `ScientificName` component has an italic font style.